xrmocap/utils/__init__.py

```diff
@@ -0,0 +1,8 @@
+# yapf: disable
+from .camera_utils import (
+    project_point_radial, project_pose, unfold_camera_param,
+)
+
+# yapf: enable
+
+__all__ = ['project_point_radial', 'project_pose', 'unfold_camera_param']
```

Comment:
Same as https://github.com/openxrlab/xrmocap/pull/12/files#r969211128..

Reply:
As offline discussion, we may skip utils for readthedocs, or only keeping the important fucntions.